spec/services/change_schedule_spec.rb

```diff
@@ -117,8 +127,8 @@
 
   context "ECT participant profile" do
     let(:cpd_lead_provider) { induction_record.cpd_lead_provider }
-    let(:schedule_identifier) { "ecf-standard-september" }
```

Comment:
why did those need to change? can we also use `schedule_identifier:,` instead of setting the same value twice?